website/docs/docs/faq.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 5
+title: FAQ
+---
+
+# Frequently asked questions
+
+**What is Sandpack?**
+
+Sandpack is an open ecosystem of components and utilities that allow you to compile and run modern frameworks in the browser. You can either use one of our predefined `components` for embedding the *CodeSandbox* experience into your projects, or you can build your own version of `sandpack`, on top of our standard components and utilities. As you walk through this guide, you will get deeper into our ecosystem.
```

Comment:
> As you walk through this guide, you will get deeper into our ecosystem.

I'm not sure if this sentence make sense in the FAQ, rather than in the docs